praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/domain/dtos/CreateMeldezeitraumDTO.java

```diff
@@ -10,7 +10,6 @@
 @Builder
 @StartDateBeforeEndDate(startDate = "startZeitpunkt", endDate = "endZeitpunkt", message = "Startdatum muss vor dem Enddatum liegen.")
 public record CreateMeldezeitraumDTO(
-        @NotNull
```

Comment:
Why remove the NotNull? Even if the frontend does not allow such things, someone with the url only could create a Meldezeitraum now with start and end null